package.json

```diff
@@ -2356,6 +2356,12 @@
 						"markdownDescription": "EXPERIMENTAL: Whether to look for custom script definitions at `dart/config/ide/flutter.json` in Bazel workspaces. Currently supported for macOS and Linux only.",
 						"scope": "window"
 					},
+					"dart.daemonPort": {
+						"type": "number",
+						"default": false,
+						"markdownDescription": "EXPERIMENTAL: Daemon port",
```

Comment:
I've (very recently) started trying to highlight settings that are not relevant to most users more clearly. Some examples in 0b2fa58db3c5e7def01e98773824c08f6fe04647. It might be worth adding a note here that this is intended for internal Google developers?